Examples/Scotty-tvOSExample/Routing/Routable+App.swift

```diff
@@ -0,0 +1,68 @@
+//
+//  Routable+App.swift
+//  Scotty-iOS
+//
+//  Created by Fernando Arocho on 1/8/19.
+//  Copyright © 2019 Bottle Rocket Studios. All rights reserved.
+//
+
+import UIKit
+import Scotty_tvOS
```

Comment:
Ideally we can tweak the module name to just be Scotty as well- Hyperspace does this for example.